### ISSUE TITLE
Update profile image via external URL or file

### ISSUE THREAD

**Issue by FingerlessGlov3s** [BookStackApp/BookStack#4095]
### API Endpoint or Feature

I would love to be able to update user's profile images via the API, either it be passing in an external image url, or file upload as a base64 encoded image, in the JSON payload.

### Use-Case

I'm trying to sync role memberships and other details between a discord server and bookstack. I've gotten the role members updating but I did have to use a SQL query to get out the discordId of the users, but the only thing I can't do now is update the profile images to match their profile image in Discord.

**Comment by ssddanbrown** [comment]
Yeah, this makes sense to add to the user create/edit endpoints, although I don't think I'd look to complicate things with a custom (base64/url) format, we'd just follow conventions of the other endpoints that support files/images by allowing upload via a `multipart/form-data` request.

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/4095
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
